Makefile

```diff
@@ -171,6 +186,12 @@ SINGLE_FILE_C_PROGRAM_SRCS := $(wildcard $(SINGLE_FILE_C_PROGRAM)/*.c)
 # Add L3 package's library file to list of sources to be compiled.
 SINGLE_FILE_C_PROGRAM_SRCS += $(L3_SRCS)
 
+# Name of LOC-generated source file.
+ifdef L3_LOC_ENABLED
+SINGLE_FILE_C_PROGRAM_GENSRC := $(SINGLE_FILE_C_PROGRAM)/$(LOC_FILENAMES)
+SINGLE_FILE_C_PROGRAM_SRCS   += $(SINGLE_FILE_C_PROGRAM_GENSRC)
+endif
```

Comment:
All dependency of generated sources and targets will be guarded under `ifdef L3_LOC_ENABLED`.

This way, non-LOC-enabled builds will not see the generator coming into play, and there should be no change in behaviour.